DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModel.cs

```diff
@@ -6,14 +6,25 @@
 
     public class SearchableAdminViewModel : BaseFilterableViewModel
     {
-        public SearchableAdminViewModel(AdminUser adminUser)
+        public SearchableAdminViewModel(AdminUser adminUser, AdminUser loggedInAdminUser)
         {
             Id = adminUser.Id;
             Name = adminUser.SearchableName;
             CategoryName = adminUser.CategoryName ?? "All";
             EmailAddress = adminUser.EmailAddress;
             IsLocked = adminUser.IsLocked;
-            IsCentreManager = adminUser.IsCentreManager;
+
+            if (loggedInAdminUser.IsUserAdmin)
+            {
+                CanShowDeactivateAdminButton = adminUser.Id != loggedInAdminUser.Id;
+            }
+            else if (loggedInAdminUser.IsCentreManager)
+            {
+                CanShowDeactivateAdminButton = !adminUser.IsUserAdmin
+                                               && !adminUser.IsCentreManager
+                                               && adminUser.Id != loggedInAdminUser.Id;
+            }
+

```

Comment:
For completeness, can we have an `else { CanShow...Button = false}`?

Reply:
I have added else block as suggested